webapp/components/form/Select/Select.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/jsx-props-no-spreading */
+import './Select.scss'
+import React from 'react'
+import PropTypes from 'prop-types'
+import ReactSelect from 'react-select'
+
+import { defaultComponents, baseStyles, baseTheme } from './config'
+import { adaptSelection } from './utils'
+
+export const Select = ({
+  customStyle = {},
+  className: customClassName,
+  components: overrideComponents,
+  options,
+  value,
+  onChange,
+  ...props
+}) => {
+  return (
+    <>
```

Comment:
the fragment is not needed

Reply:
Thanks at the beginning I had the Dropdown to compare